### ISSUE TITLE
Aria-required-parent and aria-valid-attribute-values are not present

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7229]
### Repro steps:
1. Open [UI Grid: 401 All Features (ui-grid.info)](http://ui-grid.info/docs/#!/tutorial/Tutorial:%20401%20All%20Features) CodePen.
2. Scroll down to the demo section
3. Click on the hamburger icon in top right corner table to show the options
4. Inspect using F12
5. Observe the controls does not have Aria-required-parent and aria-valid-attribute-values.

### Actual Result:
Aria-required-parent and aria-valid-attribute-values are not present

### Expected Result:
Aria-required-parent and aria-valid-attribute-values should present

### Notes:
Issue is reproducible in all the controls displayed under 'Rows'

Screenshot:
![image](https://user-images.githubusercontent.com/93735775/176796735-d2d571a9-97ee-4711-a27a-8ef1e8bb2f8d.png)

![image](https://user-images.githubusercontent.com/93735775/176796944-c79df65e-d748-48f8-b820-d18b96b0509a.png)

**Comment by msftedad** [comment]
Hi team, could you please share timeline for fixing this issue.

**Comment by msftedad** [comment]
@mportuga I've seen you add some accessibility tags on other issues - any update on this issue?

**Comment by mportuga** [comment]
@msftedad You are free to open some PRs if you believe you can fix it, I am not sure on a timeline for when I would be able to get around to it to be honest. I only work on UI-Grid part time and haven't had much free time lately, hence the gap between releases.

**Comment by msftedad** [comment]
Hi Team @mportuga , Is there any update on this issue?

**Comment by msftedad** [comment]
Hi @mportuga, Is there any progress on this issue?

**Comment by msftedad** [comment]
Hi Team @mportuga , Is there any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Please provide an update on this issue.

**Comment by msftedad** [comment]
Hi Team, Please provide an update on this issue.

**Comment by msftedad** [comment]
Hi Team, As the issue is still repro. Could you please provide an update if there is any?

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
